File: app/src/main/java/com/tianchuang/ihome_b/mvp/mvpbase/BasePresenter.java
package com.tianchuang.ihome_b.mvp.mvpbase;

/**
 * Created by Abyss on 2017/4/1.
 * description:mvp的p
 */

public interface BasePresenter<V extends BaseView> {
    void attachView(V view);
    void detachView();
    void unsubscribe();
}

File: app/src/main/java/com/tianchuang/ihome_b/mvp/BasePresenter.kt
package com.tianchuang.ihome_b.mvp

/**
 * Created by Abyss on 2017/4/1.
 * description:mvp的p
 */

interface BasePresenter<in V : BaseView> {
    fun attachView(view: V)
    fun detachView()
    //    void unsubscribe();
}
